### DISCUSSION TITLE
UC mode and Tor

### DISCUSSION THREAD

**Discussion by RealSting** [seleniumbase/SeleniumBase#3229] [General]
Hi!
I've Tor installed (Windows (x86_64) - 14.0 (tor 0.4.8.12)) as a windows service.
Here the installer --> https://www.torproject.org/download/tor

My configuration:
```
TYPE: SOCKS5
IP: 127.0.0.1
PORT: 9050
```

I don't be able to find a simple example to open a generic web page.
Any hints please? ::)

**Comment by mdmintz** [comment]
UC Mode is typically Chrome-only. There may be limited support for other Chromium browsers, which you can use by setting the binary location. See: https://github.com/seleniumbase/SeleniumBase/issues/2323#issuecomment-1830943127

**Reply by RealSting** [reply]
I use Chrome as browser.
I use SmartProxy as plugin.
Some sites are configured to connect via Tor.
Anyway is it possible to specify to use Tor proxy (Tor Windows Service is always on on my PC) on SeleniumBase?
On normal Selenium I use

```
proxy = "localhost"
port = "9050"
chrome_options.add_argument("--proxy-server=socks5://" + proxy + ":" + port)
```
Thanks for your time :)

**Reply by mdmintz** [reply]
There's a `proxy` arg in `seleniumbase`:
`proxy="server:port"`
